src/main/java/seedu/stock/logic/commands/ListBookmarkCommand.java

```diff
@@ -19,11 +22,15 @@
 
     public static final Predicate<Stock> IS_BOOKMARKED = new BookmarkedPredicate();
 
+    private static final Logger logger = LogsCenter.getLogger(ListBookmarkCommand.class);
+
 
     @Override
     public CommandResult execute(Model model) {
+        logger.log(Level.INFO, "Starting to execute list bookmark command");
         requireNonNull(model);
         model.updateFilteredStockList(IS_BOOKMARKED);
+        logger.log(Level.INFO, "Finished listing all stocks successfully");
```

Comment:
Maybe can put a bookmark after all to distinguish against list all command?

Reply:
I think that's a great idea, thanks for the suggestion! Will change now.